backend/substrapp/tasks/tasks.py

```diff
@@ -905,3 +887,25 @@ def get_subtuple_directory(subtuple):
 def get_chainkeys_directory(compute_plan_id):
     return path.join(getattr(settings, 'MEDIA_ROOT'), getattr(settings, 'ORG_NAME'), 'computeplan',
                      compute_plan_id, 'chainkeys')
+
+
+def remove_algo_images(algo_hashes):
+    client = docker.from_env()
+    for algo_hash in algo_hashes:
+        algo_docker = f'substra/algo_{algo_hash[0:8]}'.lower()
```

Comment:
What about extracting this line into a function that could also be called from `_do_task`?

Reply:
You're talkin about line 895, to make it as a function because it's also used in `_do_task` ?

Reply:
Done